.github/ISSUE_TEMPLATE/01-building-an-application.yml

```diff
@@ -54,7 +55,7 @@ body:
     id: logs
     attributes:
       label: Relevant log output
-      description: Please copy and paste any (short!) relevant log output.  Longer logs can be attached as .txt files.  This is likely a [diagnostic MSBuild log](https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/troubleshooting#diagnostic-msbuild-output) or attach a [MSBuild binlog](https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2022#save-a-binary-log).
+      description: Please copy and paste any (short!) relevant log output.  Longer logs can be attached as .txt files.  This is likely a [diagnostic MSBuild log](https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/troubleshooting#diagnostic-msbuild-output) or attach a [MSBuild binlog](https://aka.ms/binlog).
```

Comment:
If there are `docs.microsoft.com` URLs, should those be `learn.microsoft.com` now?

I think we are also supposed to remove the `/en-us/`, so it will localize for other languages.

Reply:
Correct on both counts: use learn.microsoft.com, and remove `/en-us` from the URLs.

Reply:
Fixed